libs/platform/ebpf_tracelog.c

```diff
@@ -0,0 +1,1059 @@
+    CASE_EPOCH:                                                                                       \
+        _EBPF_LOG_MESSAGE_GUID_GUID(trace_level, KEYWORD_EPOCH, message, guid1, guid2);               \
+        break;                                                                                        \
+    CASE_CORE:                                                                                        \
+        _EBPF_LOG_MESSAGE_GUID_GUID(trace_level, KEYWORD_CORE, message, guid1, guid2);                \
+        break;                                                                                        \
+    CASE_LINK:                                                                                        \
+        _EBPF_LOG_MESSAGE_GUID_GUID(trace_level, KEYWORD_LINK, message, guid1, guid2);                \
+        break;                                                                                        \
+    CASE_MAP:                                                                                         \
+        _EBPF_LOG_MESSAGE_GUID_GUID(trace_level, KEYWORD_MAP, message, guid1, guid2);                 \
+        break;                                                                                        \
+    CASE_PROGRAM:                                                                                     \
+        _EBPF_LOG_MESSAGE_GUID_GUID(trace_level, KEYWORD_PROGRAM, message, guid1, guid2);             \
+        break;                                                                                        \
+    CASE_API:                                                                                         \
+        _EBPF_LOG_MESSAGE_GUID_GUID(trace_level, KEYWORD_API, message, guid1, guid2);                 \
+        break;                                                                                        \
+    CASE_PRINTK:                                                                                      \
+        _EBPF_LOG_MESSAGE_GUID_GUID(trace_level, KEYWORD_PRINTK, message, guid1, guid2);              \
+        break;                                                                                        \
+    CASE_NATIVE:                                                                                      \
+        _EBPF_LOG_MESSAGE_GUID_GUID(trace_level, KEYWORD_NATIVE, message, guid1, guid2);              \
+        break;                                                                                        \
+    default:                                                                                          \
+        break;                                                                                        \
+    }
+__declspec(noinline) void ebpf_log_message_guid_guid(
+    ebpf_tracelog_level_t trace_level,
+    ebpf_tracelog_keyword_t keyword,
+    _In_z_ const char* message,
+    _In_ const GUID guid1,
+    _In_ const GUID guid2)
```

Comment:
```suggestion
    GUID guid1,
    GUID guid2)
```

Reply:
Changed the existing implementation to use pointers instead.